tests/Aspire.Hosting.Tests/Keycloak/AddKeycloakTests.cs

```diff
@@ -0,0 +1,155 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
@mitchdenny - do you think we should start with these tests in a separate assembly from the beginning?

Reply:
Yeah I think we had better start doing this. @julioct take a look at Aspire.Hosting.Containers.Tests ... we split out test cases for `WithDockerfile` into a separate assembly. We are planning on breaking out a lot of the tests in `Aspire.Hosting.Tests` moving forward.